tasks.py

```diff
@@ -109,7 +128,11 @@ def vscode(context):
 @task
 def nbshell(context):
     """Launch an interactive nbshell session."""
-    docker_compose(context, "run nautobot nautobot-server nbshell", pty=True)
+    command = "nautobot-server nbshell"
+    if INVOKE_LOCAL:
```

Comment:
I like where you're going with this, but I do not like that global behavior being toggled with a hidden environment variable. (FWIW I had the same concern about the `PYTHON_VER` variable).

It seems like we might be entering into a need to start [modifying the core parser arguments](http://docs.pyinvoke.org/en/stable/concepts/library.html#modifying-core-parser-arguments) to add a top-level `--local` flag or even `--python-ver` flag that can be used, but it appears that to go that route `tasks.py` would need to become an executable in its own right, OR we'd have to create our own `Task` subclass to provide default arguments. BUT that also starts adding more complexity. So I don't know...

Reply:
I have updated the invoke tasks to utilize the [invoke configuration](http://docs.pyinvoke.org/en/stable/concepts/configuration.html).  This allows us to use env vars, an invoke.yml file or command line parameters for configuration options.  I hope this relieves your concerns here.